packages/pn-pa-webapp/src/utils/notification.utility.ts

```diff
@@ -49,7 +49,7 @@ const newNotificationRecipientsMapper = (
     return {
       denomination: recipient.recipientType === RecipientType.PG ? recipient.firstName : `${recipient.firstName} ${recipient.lastName}`,
       recipientType: recipient.recipientType,
-      taxId: recipient.taxId,
+      taxId: recipient.taxId.toUpperCase(),
```

Comment:
This is causes problems, because the paymentDocuments (below in the code) is an object where the keys are the recipients tax ids.
I suggest to do the transformation into the moethod saveRecipients in reducers.ts file